spec/features/a_user_can_log_in_and_out_spec.rb

```diff
@@ -24,7 +24,7 @@
   end
 
   it "I can log out and sign back in" do
-    within('.nav-bar') do
+    within('#menuToggle') do
       click_on 'Logout'
```

Comment:
I haven't seen this before - what does it do?

Reply:
#menuToggle doesn't do anything special is just defines of our css ids for styling of the hamburger menu.